docs/build-a-runtime/index.md

```diff
@@ -89,12 +89,10 @@ The above diagram details the two high-level components of the base9 architectur
 Our frontend language is a primitive subset of JavaScript. Currently, it can only operate on integers. Let's have a look at some code:
  
  ```js
-function b9main() {
-  b9PrintString("Hello World!");
-}
+b9PrintString("Hello World!");
 ```
 
-Above is a classic program that we all know and love, Hello, World! The source code can be found in [test/hello.src]. The `b9main` function is always the entry point to a program, like `int main` in C. `b9PrintString` is a method from a tiny library of functions that will be compiled with every program. The library is called b9stdlib and can be found in [b9/js_compiler/b9stdlib.src]. The functions in b9stdlib all call base9 [primitive functions]. In the Hello, World! program, `b9main` uses `b9PrintString` from b9stdlib to write text to console.
+ `b9PrintString` is a method from a tiny library of functions that will be compiled with every program. The library is called b9stdlib and can be found in [b9/js_compiler/b9stdlib.src]. The functions in b9stdlib all call base9 [primitive functions]. In the Hello, World! program, Our script uses `b9PrintString` from b9stdlib to write text to console.
```

Comment:
It is probably not accurate to say all b9stdlib functions call primitive functions.. even though right now they do.  I think in the future we're going to change how primitive functions are implemented, so we remove our weird cffi entirely.  For now I guess its fine, just a thought.